inc/Calculator.h

```diff
@@ -0,0 +1,394 @@
+        DrawRectangle(x,y,w,h,SKYBLUE);
+        int buttonOffset = (50 - MeasureText(buttonText.c_str(),30))/2;
+        DrawText(buttonText.c_str(), buttonOffset+x, 10 + y, 30, BLACK);
+
+        if (IsMouseButtonDown(MOUSE_BUTTON_LEFT)) return true;
+            else return false;
+        }
+        
+
+    }
+
+    /*int prec(char ch) {
+	if (ch == '^')
+		return 3;
+	else if (ch == '/' || ch == '*')
+		return 2;
+	else if (ch == '+' || ch == '-')
+		return 1;
+	else
+		return -1;
+}
+int applyOp(int b, int a, char op){
+    switch(op){
+        case '+': return a + b;
+        case '-': return b - a;
+        case '*': return a * b;
+        case '/': return b / a;
+        case '^': return pow(b, a);
+    }
+}
+float evaluateExpression(std::string expressionText)
+{   
+    std::string lastOperand = "";
+    float ans = 0.0f;
+    for( int i = 0; i <= expressionText.length(); i++){
+        if(isalnum(expressionText[i])){
+            lastOperand += expressionText[i];
+        }
+        else{
+            if (lastOperand.length() > 0){
+            operands.push(std::stof(lastOperand));
+            lastOperand = "";}
+
+            if ( expressionText[i] == '('){
+                operators.push(expressionText[i]);
+            }
+            else if(expressionText[i] == ')'){
+                while (!operators.empty() && operators.top () != '('){
+                    int val2 = operands.top();
+                    operands.pop();
+                    int val1 = operands.top();
+                    operands.pop();
+                    char op = operators.top();
+                    operators.pop();
+
+                    operands.push(applyOp(val1, val2, op));
+                }
+
+                if ( !operators.empty()){
+                    operators.pop();
+                }
+            }
+            else{
+                while (!operators.empty() && prec(operators.top()) >= prec(expressionText[i])){
+                    int val2 = operands.top();
+                    operands.pop();
+                    int val1 = operands.top();
+                    operands.pop();
+                    char op = operators.top();
+                    operators.pop();
+
+                    operands.push(applyOp(val1, val2, op));
+                }
+                operators.push(expressionText[i]);
+            }
+        }
+    }
+    while (!operands.empty()){
+        int val2 = operands.top();
+        operands.pop();
+        int val1 = operands.top();
+        operands.pop();
+
+        char op = operators.top();
+        operators.pop();
+        operands.push(applyOp(val1, val2, op));
+    }
+    return operands.top();
+}*/
+
+int precedence(char op){
+    if(op == '+'||op == '-')
+    return 1;
+    if(op == '*'||op == '/')
+    return 2;
+    if(op == '^')
+    return 3;
+    return 0;
+}
+ 
+// Function to perform arithmetic operations.
+float applyOp(float a, float b, char op){
```

Comment:
Either use switch in both of these functions or do if statement. Consistency helps with readability